Bundle/CoreBundle/Resources/doc/SETUP.md

```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
-    tags:
-        - { name: twig.extension }
+framework:
+   ...
+   translator:      { fallbacks: ["%locale%"] }
+   serializer: { enable_annotations: true }
 ```
 
-- Import the Victoire config file:
+#### Add some config
 
+`app/config/config.yml`
 ```yml
 imports:
     - { resource: @VictoireCoreBundle/Resources/config/config.yml }
-```
 
-- Install Redis:
+assetic:
+    use_controller: false
+    bundles: ~
+    filters:
+        cssrewrite: ~
+        less: ~ #Victoire needs less under v3.0.0
+
+fos_user:
+    db_driver: orm
+    firewall_name: main
+    user_class: Victoire\UserBundle\Entity\User
+    from_email:
+        address: hey@troopers.email
+        sender_name: Troopers
 
-On Mac OS X:
+snc_redis:
+    clients:
+        victoire:
+            type: predis
+            alias: victoire
+            dsn: %victoire_redis_path%
+            logging: false
+
+stof_doctrine_extensions:
+    default_locale: en_US
+    orm:
+        default:
+            tree: true
+            sluggable: true
+            timestampable: true
+            softdeleteable: true
+
+victoire_core:
+    user_class: "Victoire\\UserBundle\\Entity\\User"
+    business_entity_debug: true
+    layouts:
+        defaultLayout: "Default layout"
+
+victoire_i18n:
+    available_locales:
+        - fr
+        - en
+    locale_pattern: parameter #domain
 
-```bash
-brew install redis
+    #if locale_pattern is domain, then you'll need to define the following parameter:
+    #locale_pattern_table: %locale_pattern_table%
 ```
 
-On Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-redis-on-ubuntu-16-04
+`app/config/security.yml`
+```
+security:
+    encoders:
+        Victoire\Bundle\UserBundle\Entity\User: bcrypt
+    providers:
+        fos_userbundle:
+            id: fos_user.user_provider.username
+    firewalls:
+        dev:
+            pattern: ^/(_(profiler|wdt)|css|images|js)/
+            security: false
+        main:
+            pattern: ^/
+            form_login:
+                provider: fos_userbundle
+                csrf_provider: form.csrf_provider
+                failure_path: /login
+                check_path: /login_check
+                default_target_path: /
+            logout: true
+            anonymous: true
+            switch_user: ~
+
+    access_control:
+        - { path: ^/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+```
 
-- Setup Redis:
+`app/config/services.yml`
+```
+services:
+    twig.extension.text:
+       class: Twig_Extensions_Extension_Text
+       tags:
+            - { name: twig.extension }
+
+    twig.extension.intl:
+        class: Twig_Extensions_Extension_Intl
+        tags:
+            - { name: twig.extension }
+```
 
-```yml
-snc_redis:
-   clients:
-       victoire:
-           type: predis
-           alias: victoire
-           dsn: redis://localhost
+`app/config/parameters.yml.dist`
+```
+parameters:
+    ...
+    router.request_context.host: https://troopers.agency
```

Comment:
maybe, I suspect absolute url generation in emails (when there is not request context), so not related to victoire